network_modeling/models/cray-network-hardware.yaml

```diff
@@ -170,6 +171,15 @@ network_hardware:
       - count: 1
         speed: 1
         slot: "mgmt"
+  - name: "SLINGSHOT 200G 64P"
+    vendor: "cray"
+    model: "slingshot_hsn_switch"
+    type: "switch"
+    deprecated: True
```

Comment:
is this really deprecated hardware?

Reply:
I don't think so...copy/pasta from a switch key I borrowed as a template.

Reply:
Fixed